### PR TITLE
Heroku documentation

### PR DESCRIPTION
**Pull request by lfbrock**
Open to feedback on whether we need to doc the heroku settings, since they are described well in the app install itself (see screenshot for example). I included them, but not sure if it's helpful, or if it would be better to keep the docs short since the goal is a quick + easy install. 0/5
screenshot 2016-09-06 15 56 47

### CODE REVIEW

File: source/install/heroku.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+Heroku Installation
+--------------------
+
+1. Sign up for a free account on `Heroku <https://www.heroku.com/>`_
+
+2. Click the |HerokuDeploy|_ button
+
+.. |HerokuDeploy| image:: https://www.herokucdn.com/deploy/button.svg
+.. _HerokuDeploy: https://heroku.com/deploy?template=https://github.com/mattermost/mattermost-heroku
+3. (Optional) Adjust the settings in Heroku
```
Comment (esethna) [inline source/install/heroku.rst]: Maybe just mention that these fields autocomplete but if you'd like to customize your setup then you can optionally edit these fields.

**Comment by esethna** [comment]
+1 other than my comment above, I think it's fine to detail the optional settings, the doc is not long

**Comment by lfbrock** [comment]
@esethna, updated the line to be  "(Optional) Customize the default settings in Heroku"

**Comment by jasonblais** [comment]
+1

**Pull request merged**